app/views/pages/secondary-question-banks.html.erb

```diff
@@ -33,21 +33,7 @@
 				<div class="govuk-grid-column-full">
 				  <h4 class="govuk-heading-m">Topics</h4>
             <ul class="govuk-list govuk-body govuk-!-padding-bottom-4 govuk-!-padding-top-2">
-              <% t('pages.secondary_question_banks').each do |resource|%>
-                <li class="curriculum__list--item">
-                    <h2 class="govuk-heading-s"><%= resource[:title] %></h2>
-                    <span class="govuk-!-display-inline-block govuk-!-padding-right-9">
-                      <span class="govuk-!-padding-right-6 govuk-!-font-weight-bold">Google Form:</span> 
-                      <span class="govuk-!-padding-right-6"><%= link_to "Set A", "#{resource[:google_form_set_a_link]}", class: "govuk-link ncce-link"%></span> 
-                      <span class="govuk-!-padding-bottom-0"><%= link_to "Set B", "#{resource[:google_form_set_b_link]}", class: "govuk-link ncce-link" %></span>
-                    </span>
-                    <span class="govuk-!-display-inline-block">
-                      <span class="govuk-!-padding-right-6 govuk-!-font-weight-bold">Microsoft Form:</span> 
-                      <span class="govuk-!-padding-right-6"><%= link_to "Set A", "#{resource[:microsoft_form_set_a_link]}", class: "govuk-link ncce-link"%></span> 
-                      <span><%= link_to "Set B", "#{resource[:microsoft_form_set_b_link]}", class: "govuk-link ncce-link" %></span>
-                    </span>
-                </li>
-              <% end %>
+              <%= render "secondary-question-resources", resources: t('pages.secondary_question_banks') %>
```

Comment:
In my opinion, it is more Rails-y to pass in the collection, then you won't need to wrap the partial with an `.each do` loop
Something like this:
```
render(partial: 'secondary_question_bank', collection: t('pages.secondary_question_banks'))
```
docs: https://guides.rubyonrails.org/layouts_and_rendering.html#rendering-collections

Reply:
So this default functionality in rails appears to be true for partials containing active record instances/objects opposed to view component partials

Reply:
We can use this feature with an array (see #1717)

Reply:
🤦 oops, above link was a typo. My demo commit for passing an array to a partial is on #1719

Reply:
Sorry yes resolved now thank you